app/endpoints.js

```diff
@@ -1,9 +1,11 @@
 import {
-  oidcBaseUri, orderApiUrl, organisationApiUrl, solutionsApiUrl,
+  oidcBaseUri, orderApiUrl, organisationApiUrl, solutionsApiUrl, documentApiHost,
 } from './config';
 
 const endpoints = {
   getIdentityApiHealth: () => `${oidcBaseUri}/health/ready`,
+  getDocumentApiHealth: () => `${documentApiHost}/health/ready`,
```

Comment:
👍  this reminds me I need to also include the other api to the health checks too

Reply:
I've done a thorough job 😎